packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -287,4 +292,135 @@ describe('Ceramic integration', () => {
     await ceramic1.close()
     await ceramic2.close()
   })
+
+  it('can utilize documents cache', async () => {
```

Comment:
phew.  This test is LONG and hard to follow.  Can you break it up into some smaller tests, and possibly add some comments between phases of individual tests, so it's easier to follow what is happening here?

Reply:
refactored and added more tests